exercises/00_final/precipitation_climatology.py

```diff
@@ -2,7 +2,7 @@
 
 import json
 import numpy as np
-import matplotlib.pyplot as plt
+import matplotlib.pyplot as plot
```

Comment:
So, whilst I get that `plot` is clearer, the flipside is that `plt` is the 'standard' way to import matplotlib, so I would be tempted not to change this.

It would make an interesting discussion point in the demonstration about exercising judgment in these matters, however.

Reply:
Aha, I wasn't aware of that, in that case, I think plt probably wins out, will update :)